### PR TITLE
Update vue 3 getting started sample code

### LINKED ISSUE #117: Add Vue 3 sample to getting started code sample
The `getting_started_front_end_integration_md` code sample is currently only compatible with Vue 2. We should also provide code accomplishing the same thing that works with Vue 3. This code can be added within the same sample, above or below the current content.

### PR DESCRIPTION
**Pull request by amit-ksh**
# Pull Request

## Related issue
Fixes #117 

## What does this PR do?
Update the vue 3 getting started sample code from vue 2 in README.md

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by bidoubiwa** [comment]
Hello @amit-ksh,
Thank you very much for contributing to Meilisearch :heart:.
However, I am not available on the weekend, but I will be back on Monday :blush:.

*PS: This message was sent automatically!*

**Comment by bidoubiwa** [comment]
Hey @amit-ksh ! Thanks for waiting, your PR flew under the radar! I'll be reviewing it ASAP!

**Comment by bidoubiwa** [comment]
I tried your sample out but unfortunately it does not work. Any idea why?

```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8" />
  <link rel="stylesheet"
    href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
</head>

<body>
  <div id="app" class="wrapper">
    <ais-instant-search :search-client="searchClient" index-name="movies">
      <ais-configure :hits-per-page.camel="10" />
      <ais-search-box placeholder="Search here…" class="searchbox"></ais-search-box>
      <ais-hits>
        <div slot="item" slot-scope="{ item }">
          <ais-highlight :hit="item" attribute="title" />
        </div>
      </ais-hits>
    </ais-instant-search>
  </div>
</body>

<script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-instantsearch/vue3/umd/index.js"></script>
<script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js">
</script>

<script>
  const app = Vue
    .createApp({
      data() {
        return {
          searchClient: instantMeiliSearch('http://127.0.0.1:7700')
        }
      }
    })
    .use(VueInstantSearch)
    .mount("#app");
</script>

</html>

```

**Comment by amit-ksh** [comment]
I don't know why it's working, app is importing all the packages correctly but not able to use it plugin as well as the styled components.

**Comment by bidoubiwa** [comment]
Is it working for you?

**Comment by amit-ksh** [comment]
No. In the console, warning is logging out for `meilisearch-instantsearch` package

**Comment by bidoubiwa** [comment]
😕 Tell me if you achieve to make it work. Meanwhile I put this in draft.

**Comment by amit-ksh** [comment]
OK!

**Comment by amit-ksh** [comment]
Hey @bidoubiwa! I fix the sample code, would you review it again?

**Comment by bidoubiwa** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Comment by amit-ksh** [comment]
I will definitely give a try

**Comment by amit-ksh** [comment]
vue meilisearch app

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -87,3 +87,103 @@ getting_started_front_end_integration_md: |-
...
+
+  <head>
+    <meta charset="utf-8" />
+
+    <link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
+  </head>
+
+  <body>
+    <div id="app">
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:144]:
```suggestion
    <div id="app" class="wrapper">
```

File: README.md [hunk 2]
```diff
@@ -127,39 +123,275 @@ In the `App.vue` file:
...
 <template>
-  <ais-instant-search :search-client="searchClient" index-name="steam-video-games">
-    <ais-search-box />
-    <ais-hits>
-      <template v-slot:item="{ item }">
-        <h2>{{ item.name }}</h2>
-      </template>
-    </ais-hits>
-  </ais-instant-search>
+  <header class="header">
```
Comment (bidoubiwa) [inline README.md:126]: Why does this example needed to be changed? It should be as minimal as the one for vue2 with the same style

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -87,3 +87,103 @@ getting_started_front_end_integration_md: |-
...
+
+
+  // In App.vue
+  <template>
+    <ais-instant-search :search-client="searchClient" index-name="movies">
+      <ais-search-box />
+      <ais-hits>
+        <div slot="item" slot-scope="{ item }">
+          <h2>{{ item.title }}</h2>
+        </div>
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:109]: This does not seem to work. Maybe the previous example from the README in vue3 can help you out

File: README.md [hunk 4]
```diff
@@ -127,39 +123,275 @@ In the `App.vue` file:
...
+  margin: 2rem auto;
+  text-align: center;
+}
+.ais-SearchBox-form {
+  margin-bottom: 20px;
 }
 </style>
 ```
 
+In the `DebouncedSearchBox.vue` file:
```
Comment (bidoubiwa) [inline README.md:302]: Why did you add this? Was it required?

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Comment by amit-ksh** [comment]
Made all the requested changes.

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
   - The third and final `<script>` creates a new Vue instance and instructs it to use `instant-meilisearch`
+
+
+landing_getting_started_with_vue_3: |-
+  // In main.js
+  import { createApp } from "vue";
+  import App from "./App.vue";
+  import InstantSearch from "vue-instantsearch/vue3/es";
+
+  createApp(App).use(InstantSearch).mount("#app");
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:98]:
```suggestion
  import { createApp } from 'vue';
  import App from './App.vue';
  import InstantSearch from 'vue-instantsearch/vue3/es';
  import "instantsearch.css/themes/algolia-min.css";
  import "@meilisearch/instant-meilisearch/templates/basic_search.css";

  createApp(App).use(InstantSearch).mount('#app');
```

I added the CSS imports here

File: .code-samples.meilisearch.yaml [hunk 6]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
+  
+
+  // In App.vue  
+  <template>
+    <ais-instant-search
+      :search-client="searchClient"
+      index-name="steam-video-games"
+    >
+      <ais-configure :hits-per-page.camel="10" />
+      <ais-search-box placeholder="Search here..."></ais-search-box>
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:108]:
```suggestion
      <ais-search-box />
```

File: .code-samples.meilisearch.yaml [hunk 7]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
+  createApp(App).use(InstantSearch).mount("#app");
+  
+
+  // In App.vue  
+  <template>
+    <ais-instant-search
+      :search-client="searchClient"
+      index-name="steam-video-games"
+    >
+      <ais-configure :hits-per-page.camel="10" />
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:107]:
```suggestion
```

File: .code-samples.meilisearch.yaml [hunk 8]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
+        <template v-slot:item="{ item }">
+          <ais-highlight :hit="item" attribute="name" />
+        </template>
+      </ais-hits>
+    </ais-instant-search>
+  </template>
+
+  <script>
+  import { instantMeiliSearch } from "@meilisearch/instant-meilisearch";
+  import "@meilisearch/instant-meilisearch/templates/basic_search.css";
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:119]:
```suggestion
```

File: .code-samples.meilisearch.yaml [hunk 9]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
+    },
+  };
+  </script>
+
+  <style>
+  body {
+    padding: 2rem;
+  }
+  </style>
+
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:138]:
```suggestion
```

File: .code-samples.meilisearch.yaml [hunk 10]
```diff
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
...
+  <head>
+    <meta charset="utf-8" />
+
+    <link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
+  </head>
+
+  <body>
+    <div id="app">
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:154]: Not fixed, should have wrapper as class

File: README.md [hunk 11]
```diff
@@ -138,14 +140,15 @@ In the `App.vue` file:
 </template>
 
 <script>
-import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
+import { instantMeiliSearch } from "@meilisearch/instant-meilisearch";
+import "@meilisearch/instant-meilisearch/templates/basic_search.css";
```
Comment (bidoubiwa) [inline README.md:144]: Could you add 
```
import "instantsearch.css/themes/algolia-min.css";
import "@meilisearch/instant-meilisearch/templates/basic_search.css";
```

In App.vue and remove 
```
import "@meilisearch/instant-meilisearch/templates/basic_search.css";
``` 

from here?

File: README.md [hunk 12]
```diff
@@ -154,7 +157,6 @@ export default {
 
 <style>
 body {
-  font-family: sans-serif;
   padding: 1em;
 }
 </style>
```
Comment (bidoubiwa) [inline README.md:162]: this `style` tag can be removed

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Thanks a lot for fixing these bugs so quickly 🙏

Once you're done, can you put the two code samples in a new file named
`.code-samples-vue-3.meilisearch.yaml`
?

File: README.md (hunk 12 above)
Comment (amit-ksh) [reply README.md]: Removed!

**Comment by amit-ksh** [comment]
@bidoubiwa could you review my changes please?

File: .code-samples-vue-3.meilisearch.yaml [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+        searchClient: instantMeiliSearch(
+          "https://integration-demos.meilisearch.com",
+          "99d1e034ed32eb569f9edc27962cccf90b736e4c5a70f7f5e76b9fab54d6a185"
+        ),
+      };
+    },
+  };
+  </script>
+
+
```
Comment (bidoubiwa) [inline .code-samples-vue-3.meilisearch.yaml:42]:
```suggestion
  </script>
```

Comment (bidoubiwa) [inline .code-samples-vue-3.meilisearch.yaml:42]:
```suggestion
  </script>
```

**Review by bidoubiwa — COMMENTED** [review]
After this small change I think the PR is good to be merged :)

**Comment by amit-ksh** [comment]
Done!

**Review by bidoubiwa — APPROVED** [review]
Nice work 🔥 Thanks a lot

**Comment by bidoubiwa** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**